### PR TITLE
Improving

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *congressus_ynh require a mysql galette*

## Solution

- *implement mysql*
- *apply example_ynh*
- *full install automation*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

**Comment by yalh76** [comment]
!testme

File: scripts/_common.sh [hunk 1]
```diff
@@ -7,57 +7,18 @@
 # dependencies used by the app
 YNH_PHP_VERSION="7.3"
 
-pkg_dependencies="postgresql php${YNH_PHP_VERSION}-tidy php${YNH_PHP_VERSION}-intl php${YNH_PHP_VERSION}-mbstring php${YNH_PHP_VERSION}-pgsql php${YNH_PHP_VERSION}-xml php${YNH_PHP_VERSION}-gd php${YNH_PHP_VERSION}-curl"
+pkg_dependencies="php${YNH_PHP_VERSION}-tidy php${YNH_PHP_VERSION}-intl php${YNH_PHP_VERSION}-mbstring php${YNH_PHP_VERSION}-xml php${YNH_PHP_VERSION}-gd php${YNH_PHP_VERSION}-curl"
```
Comment (ericgaspar) [inline scripts/_common.sh:10]: don't we need to add `php${YNH_PHP_VERSION}-mysql`

Comment (yalh76) [reply scripts/_common.sh]: using the current yunohost php, it's already included, but we can add it

**Comment by yalh76** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

File: scripts/backup [hunk 2]
```diff
@@ -65,6 +61,18 @@ ynh_psql_dump_db --database="$db_name" > db.sql
 
 ynh_backup --src_path="/etc/logrotate.d/$app"
 
+#=================================================
+# BACKUP THE POSTGRESQL DATABASE
+#=================================================
+ynh_print_info --message="Backing up the PostgreSQL database..."
+
+if [ $database == "pgsql" ]
+then
```
Comment (ericgaspar) [inline scripts/backup:70]:
```suggestion
then
    ynh_print_info --message="Backing up the PostgreSQL database..."
```
I will include the  `ynh_print_info` in the if statement or have it outside with generique phrasing like `Backing up the database...`

Comment (yalh76) [reply scripts/backup:69]: I included all `ynh_print_info` in the if statement

**Pull request merged**